UIKitDZ/UIKitDZ/Application/Modules/BonusModule/Presenter/BonusPresenter.swift

```diff
@@ -0,0 +1,37 @@
+// BonusPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Интерфейс взаимодействия c презентером
+protocol BonusPresenterProtocol {
+    /// Сообщает вью, что нужно закрыть экран
+    func closeScreen()
+    /// Устанавливает новый текст в лейбле вью
+    func setBonusesLabelText()
```

Comment:
```suggestion
    func  fetchBonuses()
```
мы просим презентера данные